elastica/contact_forces.py

```diff
@@ -0,0 +1,77 @@
+__doc__ = """ Numba implementation module containing contact between rods and rigid bodies and other rods rigid bodies or surfaces."""
+
+from elastica.typing import SystemType, AllowedContactType
+from elastica.rod import RodBase
+
+
+class NoContact:
+    """
+    This is the base class for contact applied between rod-like objects and allowed contact objects.
+
+    Notes
+    -----
+    Every new contact class must be derived
+    from NoContact class.
+
+    """
+
+    def __init__(self):
+        """
+
+        Parameters
+        ----------
+
+        """
+
+    def _order_check(
+        self,
+        system_one,
+        system_two,
+    ):
+        """
+        This checks the contact order between a SystemType object and an AllowedContactType object, the order should follow: Rod, Rigid body, Surface.
+        In NoContact class, this just checks if system_two is a rod then system_one must be a rod.
+
+
+        Parameters
+        ----------
+        system_one
+            SystemType
+        system_two
+            AllowedContactType
+        Returns
+        -------
```

Comment:
if nothing then skip?

Reply:
Are you talking about the Returns? If so, should I also change it in apply_torques in FreeJoint; and apply_forces and apply_torques in NoForces?

Reply:
yes whatever you decided with @armantekinalp to keep it consistent

Reply:
@Ali-7800 let's remove, I think might be breaking the documentation build.

Reply:
I created issue #287 for fixing the other functions